src/Components/AppMenu/AppMenu.tsx

```diff
@@ -45,6 +52,16 @@ export default function AppMenu() {
       });
     }
   };
+  const sendCompareGraphToServer = () => {
+    if (graph) {
+      return compareGraph({ graph })
+        .then((data) => {
+          // @ts-ignore
```

Comment:
Optional: when using // @ts-ignire state why you used it, someone might come by it later notice it and fix it...